### PR TITLE
Create migration guidance for `hidden-when-printing()`, `print-hidden()`, `when-not-printing()`, `when-printing()` functions

### LINKED ISSUE #5044: Create migration guidance for `when-printing`, `when-not-printing`, `hidden-when-printing`, and `print-hidden`  mixins

### PR DESCRIPTION
**Pull request by romellogoodman**
### WHY are these changes introduced?

Fixes #5044

### WHAT is this pull request doing?

- Add documentation for `hidden-when-printing()`, `print-hidden()`, `when-not-printing()`, `when-printing()` functions functions to our migration guide

<hr>

### Helpful References

- #4985
- #4986
- #4995

### CODE REVIEW

**Review by aaronccasanova — APPROVED** [review]

**Review by aveline — APPROVED** [review]
🎉

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 1]
```diff
@@ -321,6 +341,18 @@ padding: 0;
 list-style: none;
 ```
 
+#### `when-not-printing()`
+
+| Function                      | Replacement Value/Token |
```
Comment (lgriffee) [inline documentation/guides/migrating-from-v8-to-v9.md]:
```suggestion
| Mixin                      | Replacement Value/Token |
```
Sorry for the total nitpick 😬 just want to keep consistent with the other sections

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 2]
```diff
@@ -321,6 +341,18 @@ padding: 0;
...
+|------------------------------|-------------------------|
+| `@include when-not-printing` | `@media not print`      |
+
+#### `when-printing()`
+
+| Function                      | Replacement Value/Token |
```
Comment (lgriffee) [inline documentation/guides/migrating-from-v8-to-v9.md]:
```suggestion
| Mixin                      | Replacement Value/Token |
```
Sorry for the total nitpick 😬 just want to keep consistent with the other sections

**Review by lgriffee — APPROVED** [review]
LGTM!! 🎉🚢✨💯

**Review by alex-page — APPROVED** [review]

File: documentation/guides/migrating-from-v8-to-v9.md (hunk 1 above)
Comment (romellogoodman) [reply documentation/guides/migrating-from-v8-to-v9.md]: Added!

**Review by alex-page — APPROVED** [review]

**Comment by romellogoodman** [comment]
For transparency, force merged this PR. The Chromatic step and the changelog steps were hanging and we technically aren't observing them for this PR.

**Comment by lgriffee** [comment]
💯 That's been happening to me too! I've been noticing Chromatic hanging on some PRs and not others recently 😣 Not sure what's going on there 😢

**Pull request merged**